tests/puppeteer/campaigns/utils/initPages.js

```diff
@@ -0,0 +1,47 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+const FOBasePage = require('@pages/FO/FObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const HomePage = require('@pages/FO/home');
+const InstallPage = require('@pages/Install/install');
+
+module.exports =  async function initBasicPages(context = 'BO', pagesObjects, page) {
```

Comment:
page is required to use this function, so context and pagesObjects are by default required too, I think context doesn't need a default value, btw you can use object destructuring if you want to let peoples the liberty to use this function without calling a context

That would be something like this :

```suggestion
module.exports =  async function initBasicPages({context = 'BO', pagesObjects = {}, page}) {
```

And that would be usable like this :

```
let object = init({page});
```

and with context and adding new pages :
```
let object = init({page, context: 'BO', pagesObjects});
```